From b17bacb507c040893913e955c12d044b7cdab2a1 Mon Sep 17 00:00:00 2001
From: Moosa <moosa.baransi@babcomsoftware.com>
Date: Mon, 27 Jun 2011 15:31:17 +0300
Subject: [PATCH] wpa-gui-e: fix display issue on small screen

Makes the main wpa-gui screen scrollable to allow using it on screens
with low resolution in the "y" axis

Upstream status: pending

Signed-off-by: Moosa Baransi <moosa.baransi@babcomsoftware.com>

---
 wpa_supplicant/wpa_gui-qt4/wpagui.ui |  609 +++++++++++++++++-----------------
 1 files changed, 301 insertions(+), 308 deletions(-)

diff --git a/wpagui.ui b/wpagui.ui
index 9f9039f..9175971 100644
--- a/wpagui.ui
+++ b/wpagui.ui
@@ -1,230 +1,224 @@
-<ui version="4.0" >
+<?xml version="1.0" encoding="UTF-8"?>
+<ui version="4.0">
  <class>WpaGui</class>
- <widget class="QMainWindow" name="WpaGui" >
-  <property name="geometry" >
+ <widget class="QMainWindow" name="WpaGui">
+  <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
-    <width>345</width>
-    <height>330</height>
+    <width>365</width>
+    <height>373</height>
    </rect>
   </property>
-  <property name="windowTitle" >
+  <property name="windowTitle">
    <string>wpa_gui</string>
   </property>
-  <property name="windowIcon" >
-   <iconset resource="icons.qrc" >
+  <property name="windowIcon">
+   <iconset resource="icons.qrc">
     <normaloff>:/icons/wpa_gui.svg</normaloff>:/icons/wpa_gui.svg</iconset>
   </property>
-  <widget class="QWidget" name="widget" >
-   <layout class="QGridLayout" >
-    <item row="0" column="0" >
-     <widget class="QLabel" name="adapterLabel" >
-      <property name="text" >
+  <widget class="QWidget" name="widget">
+   <layout class="QGridLayout">
+    <item row="0" column="0">
+     <widget class="QLabel" name="adapterLabel">
+      <property name="text">
        <string>Adapter:</string>
       </property>
      </widget>
     </item>
-    <item row="0" column="1" >
-     <widget class="QComboBox" name="adapterSelect" />
+    <item row="0" column="1">
+     <widget class="QComboBox" name="adapterSelect"/>
     </item>
-    <item row="1" column="0" >
-     <widget class="QLabel" name="networkLabel" >
-      <property name="text" >
+    <item row="1" column="0">
+     <widget class="QLabel" name="networkLabel">
+      <property name="text">
        <string>Network:</string>
       </property>
      </widget>
     </item>
-    <item row="1" column="1" >
-     <widget class="QComboBox" name="networkSelect" />
+    <item row="1" column="1">
+     <widget class="QComboBox" name="networkSelect"/>
     </item>
-    <item row="2" column="0" colspan="2" >
-     <widget class="QTabWidget" name="wpaguiTab" >
-      <property name="currentIndex" >
+    <item row="2" column="0" colspan="2">
+     <widget class="QTabWidget" name="wpaguiTab">
+      <property name="currentIndex">
        <number>0</number>
       </property>
-      <widget class="QWidget" name="statusTab" >
-       <attribute name="title" >
+      <widget class="QWidget" name="statusTab">
+       <attribute name="title">
         <string>Current Status</string>
        </attribute>
-       <layout class="QGridLayout" >
-        <item row="0" column="0" colspan="5" >
-         <widget class="QFrame" name="frame3" >
-          <property name="frameShape" >
-           <enum>QFrame::NoFrame</enum>
-          </property>
-          <property name="frameShadow" >
-           <enum>QFrame::Plain</enum>
+       <layout class="QGridLayout">
+        <item row="1" column="1">
+         <widget class="QPushButton" name="disconnectButton">
+          <property name="text">
+           <string>Disconnect</string>
           </property>
-          <layout class="QGridLayout" >
-           <item row="0" column="0" >
-            <widget class="QLabel" name="statusLabel" >
-             <property name="text" >
-              <string>Status:</string>
-             </property>
-            </widget>
-           </item>
-           <item row="1" column="0" >
-            <widget class="QLabel" name="lastMessageLabel" >
-             <property name="text" >
-              <string>Last message:</string>
-             </property>
-            </widget>
-           </item>
-           <item row="2" column="0" >
-            <widget class="QLabel" name="authenticationLabel" >
-             <property name="text" >
-              <string>Authentication:</string>
-             </property>
-            </widget>
-           </item>
-           <item row="3" column="0" >
-            <widget class="QLabel" name="encryptionLabel" >
-             <property name="text" >
-              <string>Encryption:</string>
-             </property>
-            </widget>
-           </item>
-           <item row="4" column="0" >
-            <widget class="QLabel" name="ssidLabel" >
-             <property name="text" >
-              <string>SSID:</string>
-             </property>
-            </widget>
-           </item>
-           <item row="5" column="0" >
-            <widget class="QLabel" name="bssidLabel" >
-             <property name="text" >
-              <string>BSSID:</string>
-             </property>
-            </widget>
-           </item>
-           <item row="6" column="0" >
-            <widget class="QLabel" name="ipAddressLabel" >
-             <property name="text" >
-              <string>IP address:</string>
-             </property>
-            </widget>
-           </item>
-           <item row="0" column="1" >
-            <widget class="QLabel" name="textStatus" >
-             <property name="text" >
-              <string/>
-             </property>
-            </widget>
-           </item>
-           <item row="1" column="1" colspan="3" >
-            <widget class="QLabel" name="textLastMessage" >
-             <property name="text" >
-              <string/>
-             </property>
-            </widget>
-           </item>
-           <item row="2" column="1" >
-            <widget class="QLabel" name="textAuthentication" >
-             <property name="text" >
-              <string/>
-             </property>
-            </widget>
-           </item>
-           <item row="3" column="1" >
-            <widget class="QLabel" name="textEncryption" >
-             <property name="text" >
-              <string/>
-             </property>
-            </widget>
-           </item>
-           <item row="4" column="1" >
-            <widget class="QLabel" name="textSsid" >
-             <property name="text" >
-              <string/>
-             </property>
-            </widget>
-           </item>
-           <item row="5" column="1" >
-            <widget class="QLabel" name="textBssid" >
-             <property name="text" >
-              <string/>
-             </property>
-            </widget>
-           </item>
-           <item row="6" column="1" >
-            <widget class="QLabel" name="textIpAddress" >
-             <property name="text" >
-              <string/>
-             </property>
-            </widget>
-           </item>
-          </layout>
          </widget>
         </item>
-        <item row="1" column="0" >
-         <spacer>
-          <property name="orientation" >
-           <enum>Qt::Vertical</enum>
-          </property>
-          <property name="sizeHint" >
-           <size>
-            <width>20</width>
-            <height>40</height>
-           </size>
-          </property>
-         </spacer>
-        </item>
-        <item row="1" column="1" >
-         <widget class="QPushButton" name="connectButton" >
-          <property name="text" >
-           <string>Connect</string>
+        <item row="1" column="2">
+         <widget class="QPushButton" name="scanButton">
+          <property name="text">
+           <string>Scan</string>
           </property>
          </widget>
         </item>
-        <item row="1" column="2" >
-         <widget class="QPushButton" name="disconnectButton" >
-          <property name="text" >
-           <string>Disconnect</string>
+        <item row="1" column="0">
+         <widget class="QPushButton" name="connectButton">
+          <property name="text">
+           <string>Connect</string>
           </property>
          </widget>
         </item>
-        <item row="1" column="3" >
-         <widget class="QPushButton" name="scanButton" >
-          <property name="text" >
-           <string>Scan</string>
+        <item row="0" column="0" colspan="3">
+         <widget class="QScrollArea" name="scrollArea">
+          <property name="widgetResizable">
+           <bool>true</bool>
           </property>
+          <widget class="QWidget" name="scrollAreaWidgetContents">
+           <property name="geometry">
+            <rect>
+             <x>0</x>
+             <y>0</y>
+             <width>307</width>
+             <height>191</height>
+            </rect>
+           </property>
+           <layout class="QGridLayout" name="gridLayout">
+            <item row="0" column="0">
+             <widget class="QFrame" name="frame3">
+              <property name="frameShape">
+               <enum>QFrame::NoFrame</enum>
+              </property>
+              <property name="frameShadow">
+               <enum>QFrame::Plain</enum>
+              </property>
+              <layout class="QGridLayout" name="_2">
+               <item row="0" column="0">
+                <widget class="QLabel" name="statusLabel">
+                 <property name="text">
+                  <string>Status:</string>
+                 </property>
+                </widget>
+               </item>
+               <item row="1" column="0">
+                <widget class="QLabel" name="lastMessageLabel">
+                 <property name="text">
+                  <string>Last message:</string>
+                 </property>
+                </widget>
+               </item>
+               <item row="2" column="0">
+                <widget class="QLabel" name="authenticationLabel">
+                 <property name="text">
+                  <string>Authentication:</string>
+                 </property>
+                </widget>
+               </item>
+               <item row="3" column="0">
+                <widget class="QLabel" name="encryptionLabel">
+                 <property name="text">
+                  <string>Encryption:</string>
+                 </property>
+                </widget>
+               </item>
+               <item row="4" column="0">
+                <widget class="QLabel" name="ssidLabel">
+                 <property name="text">
+                  <string>SSID:</string>
+                 </property>
+                </widget>
+               </item>
+               <item row="5" column="0">
+                <widget class="QLabel" name="bssidLabel">
+                 <property name="text">
+                  <string>BSSID:</string>
+                 </property>
+                </widget>
+               </item>
+               <item row="6" column="0">
+                <widget class="QLabel" name="ipAddressLabel">
+                 <property name="text">
+                  <string>IP address:</string>
+                 </property>
+                </widget>
+               </item>
+               <item row="0" column="1">
+                <widget class="QLabel" name="textStatus">
+                 <property name="text">
+                  <string/>
+                 </property>
+                </widget>
+               </item>
+               <item row="1" column="1" colspan="3">
+                <widget class="QLabel" name="textLastMessage">
+                 <property name="text">
+                  <string/>
+                 </property>
+                </widget>
+               </item>
+               <item row="2" column="1">
+                <widget class="QLabel" name="textAuthentication">
+                 <property name="text">
+                  <string/>
+                 </property>
+                </widget>
+               </item>
+               <item row="3" column="1">
+                <widget class="QLabel" name="textEncryption">
+                 <property name="text">
+                  <string/>
+                 </property>
+                </widget>
+               </item>
+               <item row="4" column="1">
+                <widget class="QLabel" name="textSsid">
+                 <property name="text">
+                  <string/>
+                 </property>
+                </widget>
+               </item>
+               <item row="5" column="1">
+                <widget class="QLabel" name="textBssid">
+                 <property name="text">
+                  <string/>
+                 </property>
+                </widget>
+               </item>
+               <item row="6" column="1">
+                <widget class="QLabel" name="textIpAddress">
+                 <property name="text">
+                  <string/>
+                 </property>
+                </widget>
+               </item>
+              </layout>
+             </widget>
+            </item>
+           </layout>
+          </widget>
          </widget>
         </item>
-        <item row="1" column="4" >
-         <spacer>
-          <property name="orientation" >
-           <enum>Qt::Vertical</enum>
-          </property>
-          <property name="sizeHint" >
-           <size>
-            <width>20</width>
-            <height>40</height>
-           </size>
-          </property>
-         </spacer>
-        </item>
        </layout>
       </widget>
-      <widget class="QWidget" name="networkconfigTab" >
-       <attribute name="title" >
+      <widget class="QWidget" name="networkconfigTab">
+       <attribute name="title">
         <string>Manage Networks</string>
        </attribute>
-       <layout class="QGridLayout" >
-        <item row="0" column="0" colspan="5" >
-         <widget class="QListWidget" name="networkList" >
-          <property name="selectionRectVisible" >
+       <layout class="QGridLayout">
+        <item row="0" column="0" colspan="5">
+         <widget class="QListWidget" name="networkList">
+          <property name="selectionRectVisible">
            <bool>true</bool>
           </property>
          </widget>
         </item>
-        <item rowspan="2" row="1" column="0" >
+        <item row="1" column="0" rowspan="2">
          <spacer>
-          <property name="orientation" >
+          <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
-          <property name="sizeHint" >
+          <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>61</height>
@@ -232,33 +226,33 @@
           </property>
          </spacer>
         </item>
-        <item row="1" column="1" >
-         <widget class="QRadioButton" name="enableRadioButton" >
-          <property name="text" >
+        <item row="1" column="1">
+         <widget class="QRadioButton" name="enableRadioButton">
+          <property name="text">
            <string>Enabled</string>
           </property>
          </widget>
         </item>
-        <item row="1" column="2" >
-         <widget class="QPushButton" name="editNetworkButton" >
-          <property name="text" >
+        <item row="1" column="2">
+         <widget class="QPushButton" name="editNetworkButton">
+          <property name="text">
            <string>Edit</string>
           </property>
          </widget>
         </item>
-        <item row="1" column="3" >
-         <widget class="QPushButton" name="removeNetworkButton" >
-          <property name="text" >
+        <item row="1" column="3">
+         <widget class="QPushButton" name="removeNetworkButton">
+          <property name="text">
            <string>Remove</string>
           </property>
          </widget>
         </item>
-        <item rowspan="2" row="1" column="4" >
+        <item row="1" column="4" rowspan="2">
          <spacer>
-          <property name="orientation" >
+          <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
-          <property name="sizeHint" >
+          <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>61</height>
@@ -266,106 +260,106 @@
           </property>
          </spacer>
         </item>
-        <item row="2" column="1" >
-         <widget class="QRadioButton" name="disableRadioButton" >
-          <property name="text" >
+        <item row="2" column="1">
+         <widget class="QRadioButton" name="disableRadioButton">
+          <property name="text">
            <string>Disabled</string>
           </property>
          </widget>
         </item>
-        <item row="2" column="2" >
-         <widget class="QPushButton" name="addNetworkButton" >
-          <property name="text" >
+        <item row="2" column="2">
+         <widget class="QPushButton" name="addNetworkButton">
+          <property name="text">
            <string>Add</string>
           </property>
          </widget>
         </item>
-        <item row="2" column="3" >
-         <widget class="QPushButton" name="scanNetworkButton" >
-          <property name="text" >
+        <item row="2" column="3">
+         <widget class="QPushButton" name="scanNetworkButton">
+          <property name="text">
            <string>Scan</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
-      <widget class="QWidget" name="wpsTab" >
-       <attribute name="title" >
+      <widget class="QWidget" name="wpsTab">
+       <attribute name="title">
         <string>WPS</string>
        </attribute>
-       <layout class="QGridLayout" name="wpsGridLayout" >
-        <item row="0" column="0" >
-         <widget class="QLabel" name="label_2" >
-          <property name="text" >
+       <layout class="QGridLayout" name="wpsGridLayout">
+        <item row="0" column="0">
+         <widget class="QLabel" name="label_2">
+          <property name="text">
            <string>Status:</string>
           </property>
          </widget>
         </item>
-        <item row="0" column="1" colspan="3" >
-         <widget class="QLabel" name="wpsStatusText" >
-          <property name="text" >
+        <item row="0" column="1" colspan="3">
+         <widget class="QLabel" name="wpsStatusText">
+          <property name="text">
            <string/>
           </property>
          </widget>
         </item>
-        <item row="1" column="0" colspan="2" >
-         <widget class="QPushButton" name="wpsPbcButton" >
-          <property name="text" >
+        <item row="1" column="0" colspan="2">
+         <widget class="QPushButton" name="wpsPbcButton">
+          <property name="text">
            <string>PBC - push button</string>
           </property>
          </widget>
         </item>
-        <item row="2" column="0" colspan="2" >
-         <widget class="QPushButton" name="wpsPinButton" >
-          <property name="text" >
+        <item row="2" column="0" colspan="2">
+         <widget class="QPushButton" name="wpsPinButton">
+          <property name="text">
            <string>Generate PIN</string>
           </property>
          </widget>
         </item>
-        <item row="2" column="2" >
-         <widget class="QLabel" name="label" >
-          <property name="text" >
+        <item row="2" column="2">
+         <widget class="QLabel" name="label">
+          <property name="text">
            <string>PIN:</string>
           </property>
          </widget>
         </item>
-        <item row="2" column="3" >
-         <widget class="QLineEdit" name="wpsPinEdit" >
-          <property name="enabled" >
+        <item row="2" column="3">
+         <widget class="QLineEdit" name="wpsPinEdit">
+          <property name="enabled">
            <bool>false</bool>
           </property>
-          <property name="readOnly" >
+          <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
-        <item row="3" column="0" colspan="2" >
-         <widget class="QPushButton" name="wpsApPinButton" >
-          <property name="enabled" >
+        <item row="3" column="0" colspan="2">
+         <widget class="QPushButton" name="wpsApPinButton">
+          <property name="enabled">
            <bool>false</bool>
           </property>
-          <property name="text" >
+          <property name="text">
            <string>Use AP PIN</string>
           </property>
          </widget>
         </item>
-        <item row="3" column="2" >
-         <widget class="QLabel" name="label_3" >
-          <property name="text" >
+        <item row="3" column="2">
+         <widget class="QLabel" name="label_3">
+          <property name="text">
            <string>AP PIN:</string>
           </property>
          </widget>
         </item>
-        <item row="3" column="3" >
-         <widget class="QLineEdit" name="wpsApPinEdit" >
-          <property name="enabled" >
+        <item row="3" column="3">
+         <widget class="QLineEdit" name="wpsApPinEdit">
+          <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
-        <item row="4" column="0" colspan="4" >
-         <widget class="QTextEdit" name="wpsInstructions" >
-          <property name="readOnly" >
+        <item row="4" column="0" colspan="4">
+         <widget class="QTextEdit" name="wpsInstructions">
+          <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
@@ -376,149 +370,148 @@
     </item>
    </layout>
   </widget>
-  <widget class="QMenuBar" name="MenuBar" >
-   <property name="geometry" >
+  <widget class="QMenuBar" name="MenuBar">
+   <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
-     <width>345</width>
-     <height>24</height>
+     <width>365</width>
+     <height>23</height>
     </rect>
    </property>
-   <widget class="QMenu" name="fileMenu" >
-    <property name="title" >
+   <widget class="QMenu" name="fileMenu">
+    <property name="title">
      <string>&amp;File</string>
     </property>
-    <addaction name="fileEventHistoryAction" />
-    <addaction name="fileSaveConfigAction" />
-    <addaction name="actionWPS" />
-    <addaction name="actionPeers" />
-    <addaction name="separator" />
-    <addaction name="fileExitAction" />
+    <addaction name="fileEventHistoryAction"/>
+    <addaction name="fileSaveConfigAction"/>
+    <addaction name="actionWPS"/>
+    <addaction name="actionPeers"/>
+    <addaction name="separator"/>
+    <addaction name="fileExitAction"/>
    </widget>
-   <widget class="QMenu" name="networkMenu" >
-    <property name="title" >
+   <widget class="QMenu" name="networkMenu">
+    <property name="title">
      <string>&amp;Network</string>
     </property>
-    <addaction name="networkAddAction" />
-    <addaction name="networkEditAction" />
-    <addaction name="networkRemoveAction" />
-    <addaction name="separator" />
-    <addaction name="networkEnableAllAction" />
-    <addaction name="networkDisableAllAction" />
-    <addaction name="networkRemoveAllAction" />
+    <addaction name="networkAddAction"/>
+    <addaction name="networkEditAction"/>
+    <addaction name="networkRemoveAction"/>
+    <addaction name="separator"/>
+    <addaction name="networkEnableAllAction"/>
+    <addaction name="networkDisableAllAction"/>
+    <addaction name="networkRemoveAllAction"/>
    </widget>
-   <widget class="QMenu" name="helpMenu" >
-    <property name="title" >
+   <widget class="QMenu" name="helpMenu">
+    <property name="title">
      <string>&amp;Help</string>
     </property>
-    <addaction name="helpContentsAction" />
-    <addaction name="helpIndexAction" />
-    <addaction name="separator" />
-    <addaction name="helpAboutAction" />
+    <addaction name="helpContentsAction"/>
+    <addaction name="helpIndexAction"/>
+    <addaction name="separator"/>
+    <addaction name="helpAboutAction"/>
    </widget>
-   <addaction name="fileMenu" />
-   <addaction name="networkMenu" />
-   <addaction name="helpMenu" />
+   <addaction name="fileMenu"/>
+   <addaction name="networkMenu"/>
+   <addaction name="helpMenu"/>
   </widget>
-  <action name="fileEventHistoryAction" >
-   <property name="text" >
+  <action name="fileEventHistoryAction">
+   <property name="text">
     <string>Event &amp;History</string>
    </property>
   </action>
-  <action name="fileSaveConfigAction" >
-   <property name="text" >
+  <action name="fileSaveConfigAction">
+   <property name="text">
     <string>&amp;Save Configuration</string>
    </property>
-   <property name="shortcut" >
+   <property name="shortcut">
     <string>Ctrl+S</string>
    </property>
   </action>
-  <action name="fileExitAction" >
-   <property name="text" >
+  <action name="fileExitAction">
+   <property name="text">
     <string>E&amp;xit</string>
    </property>
-   <property name="shortcut" >
+   <property name="shortcut">
     <string>Ctrl+Q</string>
    </property>
   </action>
-  <action name="networkAddAction" >
-   <property name="text" >
+  <action name="networkAddAction">
+   <property name="text">
     <string>&amp;Add</string>
    </property>
   </action>
-  <action name="networkEditAction" >
-   <property name="text" >
+  <action name="networkEditAction">
+   <property name="text">
     <string>&amp;Edit</string>
    </property>
   </action>
-  <action name="networkRemoveAction" >
-   <property name="text" >
+  <action name="networkRemoveAction">
+   <property name="text">
     <string>&amp;Remove</string>
    </property>
   </action>
-  <action name="networkEnableAllAction" >
-   <property name="text" >
+  <action name="networkEnableAllAction">
+   <property name="text">
     <string>E&amp;nable All</string>
    </property>
   </action>
-  <action name="networkDisableAllAction" >
-   <property name="text" >
+  <action name="networkDisableAllAction">
+   <property name="text">
     <string>&amp;Disable All</string>
    </property>
   </action>
-  <action name="networkRemoveAllAction" >
-   <property name="text" >
+  <action name="networkRemoveAllAction">
+   <property name="text">
     <string>Re&amp;move All</string>
    </property>
   </action>
-  <action name="helpContentsAction" >
-   <property name="enabled" >
+  <action name="helpContentsAction">
+   <property name="enabled">
     <bool>false</bool>
    </property>
-   <property name="text" >
+   <property name="text">
     <string>&amp;Contents...</string>
    </property>
   </action>
-  <action name="helpIndexAction" >
-   <property name="enabled" >
+  <action name="helpIndexAction">
+   <property name="enabled">
     <bool>false</bool>
    </property>
-   <property name="text" >
+   <property name="text">
     <string>&amp;Index...</string>
    </property>
   </action>
-  <action name="helpAboutAction" >
-   <property name="text" >
+  <action name="helpAboutAction">
+   <property name="text">
     <string>&amp;About</string>
    </property>
   </action>
-  <action name="actionWPS" >
-   <property name="enabled" >
+  <action name="actionWPS">
+   <property name="enabled">
     <bool>false</bool>
    </property>
-   <property name="text" >
+   <property name="text">
     <string>&amp;Wi-Fi Protected Setup</string>
    </property>
   </action>
-  <action name="actionPeers" >
-   <property name="text" >
+  <action name="actionPeers">
+   <property name="text">
     <string>&amp;Peers</string>
    </property>
   </action>
  </widget>
- <layoutdefault spacing="6" margin="11" />
- <pixmapfunction></pixmapfunction>
+ <layoutdefault spacing="6" margin="11"/>
  <includes>
-  <include location="global" >qtimer.h</include>
-  <include location="global" >qsocketnotifier.h</include>
-  <include location="local" >wpamsg.h</include>
-  <include location="local" >eventhistory.h</include>
-  <include location="local" >scanresults.h</include>
-  <include location="local" >peers.h</include>
+  <include location="global">qtimer.h</include>
+  <include location="global">qsocketnotifier.h</include>
+  <include location="local">wpamsg.h</include>
+  <include location="local">eventhistory.h</include>
+  <include location="local">scanresults.h</include>
+  <include location="local">peers.h</include>
  </includes>
  <resources>
-  <include location="icons.qrc" />
+  <include location="icons.qrc"/>
  </resources>
  <connections/>
 </ui>
--
1.7.0.4
